src/test/java/com/training/apitests/CatApiTests.java

```diff
@@ -0,0 +1,81 @@
+package com.training.apitests;
+
+import com.training.utilities.JsonReaderUtility;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.json.simple.JSONObject;
+import org.junit.jupiter.api.Assertions;
+import java.util.List;
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+
+public class CatApiTests {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+    protected Map<String, String> map = new JsonReaderUtility().getMap();
+    RequestSpecification httpRequest;
+
+    @BeforeEach
+    public void setup() {
+        RestAssured.baseURI = map.get("baseUri");
+        httpRequest = RestAssured.given();
+        httpRequest.header(map.get("headerName"), map.get("headerValue"));
+    }
+
+    @ParameterizedTest
+    @CsvFileSource(resources = "/getIdResponse.csv")
+    public void testGetIdResponse(String name, String id) {
+
+        Response response = httpRequest.get("/v1/breeds");
+
+        Assertions.assertEquals(response.getStatusCode(), 200);
+
+        List<String> breedName = response.jsonPath().getList("name");
+        List<String> breedId = response.jsonPath().getList("id");
+        logger.info(breedId.get(breedName.indexOf(name)));
+
+        Assertions.assertEquals(breedId.get(breedName.indexOf(name)), id);
```

Comment:
Lets use a more robust logic instead of using 2 lists.

Use a for loop to iterate over all the breeds -> and check if the breed-name equals to the one you are giving -> and if its equal then take the id of that breed